ui/components/Filters.js

```diff
@@ -233,12 +233,13 @@ function MesheryFilters({
       );
     }
 
-    if (type === "upload") {
+    if (type === "upload" || type === 'urlupload') {
+      const body = type==="upload" ? JSON.stringify({ filter_data : { filter_data : data }, save : true }) : JSON.stringify({ url : data, save : true });
```

Comment:
This is good but I think it'd be better if we can implement it with the Builder design pattern in mind, like
```javascript
let body = { save: true }

if (type === "upload") {
  body = { ...body, filter_data: {filter_data: data}}
}
if (type === "urlupload") {
  body = { ...body, url: data }
}
```
What do you think? It might be an overkill

Reply:
Haha, this looks better tbh! Will update the PR.

Reply:
This looks better tbh, will update the PR!